mitodl/micromasters
# Remove deprecated object handling code for Celery tasks

#### What are the relevant tickets?
Fixes #2718 
Fixes #2171

#### What's this PR do?
After a release puts the new tasks in place, this code can remove the logic to handle objects in Celery tasks. 

#### How should this be manually tested?
Update your profile and save it. The change should be reflected in the learner search
